packages/documentation/src/pages/platform/tools/downtime-notifications.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: Va.gov downtime notifications
+tags: pagerduty,maintenance windows,scheduled downtime
+---
+
+# Downtime Notifications
+Downtime Notications are a mechanism for handling outages of external services in the consuming front-end applications. 
+
+## Overview
+- Available as a React component in [`vets-website`](https://github.com/department-of-veterans-affairs/vets-website/blob/cb1d8072a94b8847e0af0c697ff879b42db354f7/src/platform/monitoring/DowntimeNotification/index.jsx).
```

Comment:
I asked this on another PR, I'm not sure if we want to link to the current master version of files or a specific commit like this. I typically do the former, but open to other opinions. We should probably be consistent about it.

Reply:
I changed it to just link directly to master because I think it makes sense here, but I like linking to specific commits because we don't have to worry about the file moving or a line of code moving if we're linking to a specific line number.